prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -22,6 +23,17 @@ interface SettingsProvider {
     fun findSender(fullName: String): Sender?
 
     fun findOrganizationAndReceiver(fullName: String): Pair<Organization, Receiver>?
+
+    // TODO: https://github.com/CDCgov/prime-reportstream/issues/8659
+    // this should be removed after all setting keys have been migrated to the organization
+    // it is just a temporary helper function for consolidating the keys between organizations and senders
+    fun getKeys(fullName: String): List<JwkSet>? {
```

Comment:
Can you add kdoc here?
   It looks like currently this works for:
 `getKeys("simple_report.csvuploader")`
but not for:
`getKeys("simple_report")`

But I'm thinking it would make more sense if we allow the user to pass in an `issuer` that is just the `orgName` only.   (See my other comments on `issuer` in this review)

Reply:
Yes, I can add doc as fullName in this case corresponds to a sender.  I was viewing this as a temporary function just to support grabbing keys from the two spots and not changing how issuer is treated.